domain/domain.go

```diff
@@ -326,6 +326,10 @@ func canSkipSchemaCheckerDDL(tp model.ActionType) bool {
 
 // InfoSchema gets the latest information schema from domain.
 func (do *Domain) InfoSchema() infoschema.InfoSchema {
+	if do.infoCache == nil {
```

Comment:
If mockContext can return a valid infoschema, can we revert this line?

Reply:
It is still needed as some tests does not init domain propoerly.